clusterloader2/pkg/measurement/util/histogram.go

```diff
@@ -60,3 +113,16 @@ func ConvertSampleToBucket(sample *model.Sample, h *HistogramVec) {
 	}
 	hist.Buckets[string(sample.Metric["le"])] += int(sample.Value)
 }
+
+// ConvertSampleToHistogram converts prometheus sample into Histogram.
+func ConvertSampleToHistogram(sample *model.Sample, h *Histogram) {
+	labels := make(map[string]string)
+	for k, v := range sample.Metric {
+		if k != "le" {
```

Comment:
what is "le"? define a constant inside the function for it to better understand what it represents.

Reply:
"le" is a label that  defines the upper bound of a bucket of a histogram ("le" -> "less or equal").  it define in  `github.com/prometheus/common/model` , i will change to use `model.BucketLabel`.